library/src/scripts/components/forms/formElementStyles.ts

```diff
@@ -58,6 +62,19 @@ export function formElementsVariables(theme?: object) {
         ...themeVars.subComponentStyles("colors"),
     };
 
+    const errors = {
+        bg: color("#FFF3D4"),
```

Comment:
Use `feedbackColors` from globalStyleVars

Reply:
Ok I've done so but needed to update them. Luckily they've been made consistent already in all mockups so I've used those.